src/main/java/org/nuxeo/ecm/csv/CSVImporter.java

```diff
@@ -43,4 +46,18 @@ String launchImport(CoreSession session, String parentPath, File csvFile, String
     List<CSVImportLog> getLastImportLogs(String id, int max, Status... status);
 
     CSVImportResult getImportResult(String id);
+
+    /**
+     * Returns the accepted types for file upload.
+     * 
+     * @since 7.3
+     */
+    Set<String> getAcceptedTypes();
+
+    /**
+     * Prepares an uploaded file, before the actual import.
```

Comment:
Please explain in the Javadoc what "preparing" involves, and what has to be done after the prepare phase.